Thalia-barrera-advice.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="css\thalia-barrera.css">
```

Comment:
Let's change `href="css\thalia-barrera.css"` to `href="css/thalia-barrera.css"` (forward slash between 'css' and 'thalia-barrera.css') :)